nemo/collections/asr/modules/rnnt.py

```diff
@@ -302,7 +302,7 @@ def score_hypothesis(
 
     def initialize_state(self, y: torch.Tensor) -> List[torch.Tensor]:
         batch = y.size(0)
-        state = [torch.ones([batch, self.context_size], dtype=y.dtype, device=y.device) * self.blank_idx]
+        state = [torch.ones([batch, self.context_size], dtype=torch.long, device=y.device) * self.blank_idx]
```

Comment:
Are you sure this should be long ?

Reply:
Yes, for the stateless decoder, the "state" keeps track of the last words, represented with integers.